CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Added fixture entry to `components.yaml` (requires mepo v1.23.0 or higher)
+- Added an external grid and clock setter (for NUOPC).
```

Comment:
Whoops. Looks like Git decided to add this to 2.3.5 in a merge maybe. @WilliamJamieson Can you move this up to the Unreleased section.

Reply:
Okay I fixed this